mobile-app/pubspec.lock

```diff
@@ -832,6 +839,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "4.0.0"
+  riverpod:
```

Comment:
What does this package do for us?

Reply:
wanted to use state notifier with unprovided  scope in riverpod which does not depend on context in saving the articles and 
podcasts in state.

First to stop reloading  whenever you go to the articles page and also podcasts.
We can load the data from an isolate then save them in the state using riverpod.

To reduce the lag in apps mostly in low end devices

Reply:
We will need to improve the performance of the app cause right now its very laggy.
With riverpod we can load data like articles, podcasts and save them in the state without affecting the ui.This will improve performance.
Also we can use isolate which will work with riverpod to save data to the state which will be much faster  and better.

I cant implement it cause i have no keys